src/components/ui/Chakra.jsx

```diff
@@ -0,0 +1,436 @@
+import React, { useState } from 'react';
+import {
+	Avatar,
+	Box,
+	Button,
+	Center,
+	ChakraProvider,
+	Flex,
+	Heading,
+	Input,
+	extendTheme,
+	Link,
+	List,
+	ListItem,
+	ListIcon,
+	NumberInput,
+	NumberInputField,
+	NumberInputStepper,
+	NumberIncrementStepper,
+	NumberDecrementStepper,
+	Select,
+	Tag,
+	Text,
+	Textarea,
+} from '@chakra-ui/react';
+import {
+	AddIcon,
+	DeleteIcon,
+	SearchIcon,
+	StarIcon,
+	SunIcon,
+} from '@chakra-ui/icons';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
```

Comment:
Isn't the CSS for DatePicker imported as part of DatePicker?

Reply:
No, unfortunately not.